OneHourOneLife/pelican-egg-one-hour-one-life.json

```diff
@@ -0,0 +1,74 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PANEL",
+    "meta": {
+        "version": "PLCN_v1",
+        "update_url": null
+    },
+    "exported_at": "2025-10-26T22:15:03+10:00",
+    "name": "One Hour One Life",
+    "author": "peter.game.servers@gmail.com",
+    "uuid": "f3b89fc7-7e7d-4451-a26d-062ed4920103",
+    "description": "Dedicated server for One Hour One Life\r\nhttp:\/\/onehouronelife.com\/",
+    "features": null,
+    "docker_images": {
+        "Debian": "ghcr.io\/parkervcp\/yolks:debian"
+    },
+    "file_denylist": [],
+    "startup": "cd OneLife\/server && .\/OneLifeServer",
+    "config": {
+        "files": "{\r\n    \"OneLife\/server\/settings\/port.ini\": {\r\n        \"parser\": \"file\",\r\n        \"find\": {\r\n            \"\": \"{{server.allocations.default.port}}\"\r\n        }\r\n    },\r\n    \"OneLife\/server\/settings\/clientPassword.ini\": {\r\n        \"parser\": \"file\",\r\n        \"find\": {\r\n            \"\": \"{{server.environment.PASSWORD}}\"\r\n        }\r\n    },\r\n    \"OneLife\/server\/settings\/requireClientPassword.ini\": {\r\n        \"parser\": \"file\",\r\n        \"find\": {\r\n            \"\": \"{{server.environment.REQUIRE_PASSWORD}}\"\r\n        }\r\n    },\r\n    \"OneLife\/server\/settings\/serverID.ini\": {\r\n        \"parser\": \"file\",\r\n        \"find\": {\r\n            \"\": \"{{server.environment.SERVER_NAME}}\"\r\n        }\r\n    }\r\n}",
+        "startup": "{\r\n    \"done\": \"Listening for connection on port\"\r\n}",
+        "logs": "{}",
+        "stop": "^C"
+    },
+    "scripts": {
+        "installation": {
+            "script": "# Update\r\napt update\r\napt install -y build-essential git\r\n\r\n# Create Server folder\r\nmkdir -p \/mnt\/server\/\r\ncd \/mnt\/server\/\r\n\r\n# Clone the server builder\r\ncurl -o serverPullAndBuildLinux.sh https:\/\/raw.githubusercontent.com\/jasonrohrer\/OneLife\/master\/server\/installYourOwnServer\/serverPullAndBuildLinux.sh\r\n\r\n# Run it\r\nchmod +x serverPullAndBuildLinux.sh\r\n.\/serverPullAndBuildLinux.sh\r\n\r\necho -e \"Install Complete\"",
```

Comment:
as this script almost never changes, please just paste the contense of the script as the egg install script